.eslintrc.json

```diff
@@ -1,9 +1,12 @@
 {
     "env": {
         "browser": true,
-        "es6": true
+        "es6": true,
+        "jest": true
     },
-    "extends": ["airbnb"],
+    "extends": [
+        "airbnb"
+    ],
     "globals": {
```

Comment:
Not sure about sourceType, but believe Atomics and SharedArrayBuffer isn't necessary?

Reply:
That was auto-generated by eslint